src/domains/sign/landing.tf

```diff
@@ -0,0 +1,67 @@
+data "azurerm_resource_group" "core_ext" {
+  name = format("%s-rg-external", local.product)
+}
+
+data "azurerm_resource_group" "core_rg_common" {
+  name = format("%s-rg-common", local.product)
+}
+
+data "azurerm_key_vault" "core_kv_common" {
+  name                = format("%s-kv-common", local.product)
+  resource_group_name = data.azurerm_resource_group.core_rg_common.name
+}
+
+data "azurerm_dns_zone" "io_italia_it" {
+  name                = "io.italia.it"
+  resource_group_name = data.azurerm_resource_group.core_ext.name
+}
```

Comment:
Whi not use `io_pagopa_it`?
by the way already defined [here](https://github.com/pagopa/io-infra/blob/92ac52d70476eadac24f14f581776bb441c726f8/src/core/dns_io_pagopa_it.tf#L1)

Reply:
We are requested to use `io.italia.it`
But the way, even the italia zone is already defined [here](https://github.com/pagopa/io-infra/blob/92ac52d70476eadac24f14f581776bb441c726f8/src/core/dns_io_italia_it.tf#LL1C1-L1C1)... this line is just referencing it because it is defined in a different state file.